src/binary-reader-interpreter.cc

```diff
@@ -395,8 +395,7 @@ static Result on_import(uint32_t index,
                         StringSlice field_name,
                         void* user_data) {
   Context* ctx = static_cast<Context*>(user_data);
-  assert(index < ctx->module->defined.imports.size);
-  InterpreterImport* import = &ctx->module->defined.imports.data[index];
+  InterpreterImport* import = &ctx->module->imports[index];
```

Comment:
Yeah, I figured the assert is kind of nice, but it's pretty rare to see people use .at() over operator[].